packages/termux-keyring/build.sh

```diff
@@ -1,20 +1,40 @@
 TERMUX_PKG_HOMEPAGE=https://github.com/termux
 TERMUX_PKG_DESCRIPTION="GPG public keys for the official Termux repositories"
 TERMUX_PKG_LICENSE="Apache-2.0"
-TERMUX_PKG_VERSION=1.5
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=3.12
+TERMUX_PKG_REVISION=1
```

Comment:
If we're raising the package version no revision should be added.

Reply:
It's just to get identical version and revision with master branch, so that it is a bit easier to see at exactly what point we synced the package. Without revision it will be harder to tell later if all the .gpg files correspond to 3.12-0, 3.12-1 or 3.13-2 etc (if there will be more revisions)